README.md

```diff
@@ -1097,16 +1097,16 @@ description: Exclude files from linting
 -->
 ### Filter linted files
 
-If you need to lint only a folder or exclude some files from linting, you can use optional environment parameters `FILTER_REGEX_INCLUDE` and `FILTER_REGEX_EXCLUDE`
-You can apply filters to a single linter by defining variable `<LINTER_KEY>_FILTER_REGEX_INCLUDE` and `<LINTER_KEY>_FILTER_REGEX_EXCLUDE`
+If you need to lint only a folder or exclude some files from linting, you can use the optional environment parameters `FILTER_REGEX_INCLUDE` and `FILTER_REGEX_EXCLUDE`.
+You can apply filters to a single linter by defining the variables `<LINTER_KEY>_FILTER_REGEX_INCLUDE` and `<LINTER_KEY>_FILTER_REGEX_EXCLUDE`.
 
 Examples:
 
 - Lint only src folder: `FILTER_REGEX_INCLUDE: (src/)`
 - Don't lint files inside test and example folders: `FILTER_REGEX_EXCLUDE: (test/|examples/)`
 - Don't lint javascript files inside test folder: `FILTER_REGEX_EXCLUDE: (test/.*\.js)`
```

Comment:
There is an escape sequence shown in dark red here. Is it correctly written for markdown escaping (there's already backticks)